thunderscope.py

```diff
@@ -242,9 +242,13 @@ def __init__(self, toolchain="vivado", variant="a100t"):
             "write_cfgmem -force -format mcs -size 32 -interface SPIx4 -loadbit \"up 0x00980000 {build_name}.bit\" {build_name}_update.mcs"
         ]
 
-    def create_programmer(self, name='openocd', variant="a100t"):
-        if name == 'openocd':
-            return OpenOCD("openocd_xc7_ft232.cfg", self.device[variant]["flash"])
+    def create_programmer(self, name='openfpgaloader', variant="a100t"):
```

Comment:
Can we also add the `cable` parameter as an arg to this function so we can pass it in from cli?

Reply:
We can add this in a future commit.  Accepted as-is